monitor.py

```diff
@@ -665,6 +667,12 @@ def parse_entry(self, line: str):
 
                 self.state['Cargo'].update({self.canonicalise(x['Name']): x['Count'] for x in clean})
 
+            elif event_type == 'NavRoute':
+                # Added in ED 3.7 - multi-hop route details in Route.json
+                with open(join(self.currentdir, 'NavRoute.json'), 'rb') as rf:  # type: ignore
+                    entry = json.load(rf)
```

Comment:
Dash of defensive programming, just in case.
```suggestion
                    try:
                        entry = json.load(rf)
                    except JsonDecodeError as e:
                        self.state['Route'] = None
                        logger.exception(f'Failed to read JSON from NavRoute.json: {e}')
                    else:
                        self.state['Route'] = entry
```